examples/WebInterface/esp8266-AC-control.ino

```diff
@@ -0,0 +1,259 @@
+#include <FS.h>
+#include <ESP8266WiFi.h>
+#include <ESP8266mDNS.h>
+#include <WiFiUdp.h>
+#include <ESP8266HTTPUpdateServer.h>
+#include <ESP8266WebServer.h>
+#include <WiFiManager.h>
+#include <ArduinoJson.h>
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+
+//// ###### User configuration space for AC library classes ##########
+
+#include <ir_Coolix.h> // replace library based on your AC unit model, check https://github.com/markszabo/IRremoteESP8266
+
+#define auto_mode kCoolixAuto
```

Comment:
To be consistent with other arduino code (and our examples) you should make all these #define's CAPITALISED throughout the code.

e.g.
```
#define AUTO_MODE kCoolixAuto
#define COOL_MODE kCoolixCool
```